lenses/baselens.py

```diff
@@ -467,6 +467,32 @@ def __repr__(self):
         return 'GetitemLens({!r})'.format(self.key)
 
 
+class GetitemOrElseLens(GetitemLens):
+    '''A lens that focuses an item inside a container. Analogous to
+    `dict.get`.
+
+        >>> from lenses import lens
+        >>> lens().get_('foo')
+        Lens(None, GetitemOrElseLens('foo'))
+        >>> lens({'foo': 'bar'}).get_('baz').get()
+        >>> lens({'foo': 'bar'}).get_('baz', []).get()
+        []
+        >>> from collections import OrderedDict
+        >>> lens(OrderedDict({'foo': 'bar'})).get_('baz').set('qux')
+        OrderedDict([('foo', 'bar'), ('baz', 'qux')])
```

Comment:
I initially used a regular dict, but that failed the output on python3.3 because order changed :(